ansible_rulebook/action/run_job_template.py

```diff
@@ -113,6 +113,12 @@ async def _post_process(self) -> None:
             "url": self._controller_job_url(),
             "matching_events": self.helper.get_events(),
         }
+        a_log["job_results_url"] = (
```

Comment:
@jshimkus-rh Wouldn't the a_log["url"] suffice here, it already has the job URL, we could just log the a_log or pick the a_log["url"] if we done have an error.

Reply:
Yes, it would.  I overlooked it.